docs/design/autoware-architecture/index.md

```diff
@@ -0,0 +1,43 @@
+# Architecture Overview
+
+This page describes the architecture of Autoware.
+
+## Introduction
+
+The current Autoware is defined to be a layered architecture that clarifies each module's role and simplifies the interface between them. By doing so:
+
+- Autoware's internal processing becomes more transparent
+- Collaborative development is made easier because of the reduced interdependency between modules
+- Users can easily replace an existing module (e.g. localization) with their own software component by simply wrapping their software to fit in with Autoware's interface
+
+Note that the initial focus of this architecture design was solely on driving capability, and so the following features were left as future work:
+
+- Fail safe
+- HMI
+- Real-time processing
+- Redundant system
+- State monitoring system
+
+## High-level Architecture Design
+
+![Overview](image/AutowareArchitectureOverview.drawio.svg)
```

Comment:
```suggestion
![Overview](image/autoware-architecture-overview.drawio.svg)
```

Reply:
done in [f2bda3a](https://github.com/autowarefoundation/autoware-documentation/pull/86/commits/f2bda3aa760dbd6625e00c61e8a35884cfcd48e4)